hm_pyhelper/hardware_definitions.py

```diff
@@ -357,6 +357,26 @@ def is_raspberry_pi():
         'CONTAINS_IC_IDS': []
         },
 
+    # Smart Mimic / Mimiq Finestra
+    'COMP-FINESTRA': {
+        'FRIENDLY': 'Finestra Miner',
+        'SPIBUS': 'spidev0.0',
+        'KEY_STORAGE_BUS': '/dev/i2c-1',
+        'RESET': 17,
+        'MAC': 'wlan0',
+        'STATUS_TYPE': 'RGB',
+        'STATUS': 20,
+        'RGB_LED': [20, 26, 7],
```

Comment:
What about using a dict and deprecating the old `STATUS` key. So add something like:


```
'STATUS_LED': {
  TYPE: 'RGB',
  RGB: [...]
}
```

Reply:
Ah yeah that's probably a better idea.

The only reason for keeping the old `STATUS` was to retain backward compatibility (discussed here https://github.com/NebraLtd/hm-pyhelper/issues/101#issuecomment-1018779465) and not have to update existing codebase in hm-config for example.

However, perhaps that's a bit of a lazy way to do it and it's better to move to a more understandable and future proofed way.

Reply:
Cool. And when I say 'deprecate' I don't mean that we need to change the existing logic that relies on `STATUS`. Just add the new key and switch things over gradually.

Reply:
Ah so you mean have both for a time? Like STATUS AND STATUS_LED?

Reply:
Exactly. We're doing the same thing with `/initFile.txt`. We are duplicating all the old terse keys like `BT` with human readable ones like `bluetooth` and slowly switching the logic over.

Reply:
Sounds like a plan 👌

Reply:
Take a look at the update I just made @marvinmarnold and see what you think

Reply:
I'm actually going to change `SINGLE_FALLBACK` to `SINGLE_LED` because in some cases it won't be a fallback. 

But other than that

Reply:
Some additional things to consider:
- Distinction between GPIO pins vs numbers.
- Consistently include `GPIO` as prefix or postfix. For ROCK Pi, [it's a prefix](https://github.com/NebraLtd/hm-pyhelper/blob/8107035935ce5dbb4ba7d269ec94e79a9b645f15/hm_pyhelper/hardware_definitions.py#L208).

So maybe:

```
        'STATUS_LED': {
            'TYPE': 'RGB',
            'GPIO_PINS_RGB': [20, 26, 7],
            'GPIO_PIN_SINGLE': 20
         },
```

Don't care if we switch to postfix (eg `RGB_GPIO_PINS`), but let's just make sure to use this as the new precedent.

Reply:
@marvinmarnold is the consensus to use GPIO_PINS_RGB for the physical pin numbers, but GPIO_NUMBER_RGB for the GPIO? These are GPIO20, 26 and 07 not the physical pins

Reply:
Ah, `GPIO_NUMBER_` is better then.